### PR TITLE
Add changelog for 3.4.0

### PR DESCRIPTION
**Pull request by will-moore**
cc @jburel

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,11 @@
+3.3.4 (March 2021)
+-------------------
+
+- Fix 2-column layout for a single category [#74](https://github.com/ome/omero-gallery/pull/74)
+- Filter studies by Group [#75](https://github.com/ome/omero-gallery/pull/75)
+- Clear searches button [#77](https://github.com/ome/omero-gallery/pull/77)
+- Default UI uses configured viewer [#74](https://github.com/ome/omero-gallery/pull/74)
```
Comment (jburel) [inline CHANGELOG.md:7]: That's 83 I think

Comment (will-moore) [reply CHANGELOG.md]: oops - thanks. fixed.

**Comment by jburel** [comment]
Add https://github.com/ome/omero-gallery/pull/69?

**Comment by will-moore** [comment]
I don't think #69 is of any interest to our users. That's just internal OME documentation.

**Comment by jburel** [comment]
changelog is not solely for users. We have added for example in other repo replacement of travis by GHA

**Comment by will-moore** [comment]
OK - added.

**Pull request merged**